README.md

```diff
@@ -446,6 +424,75 @@ $notice = AdminNotices::show('my_notice', 'This is a notice')
     ->notDismissible();
 ```
 
+## Custom Notices
+
+Sometimes you want to display a notice, but you want to completely style it yourself. This is
+possible and pretty straightforward to do.
+
+Start by using the `custom` method on the notice. This will disable all standard visual and behavior
+
+```php
+use StellarWP\AdminNotices\AdminNotices;
+
+$notice = AdminNotices::show('my_notice_custom', 'This is a notice')
+    ->custom();
+```
+
+### `location()`
+
+***Default:*** 'standard'
+
+Sets the location in one of the standard WordPress notice locations. By default, the notice will be
+displayed in the standard location.
+
+Parameters:
+
+1. `string $location` - The location to display the notice. Can be 'standard', 'above_header', '
+   below_header', or 'inline'. Note that 'standard' and 'below_header' are the same location.
+
+```php
+use StellarWP\AdminNotices\AdminNotices;
+
+// Display the notice above the header
+$notice = AdminNotices::show('my_notice_custom', 'This is a notice')
+    ->custom()
+    ->location('above_header');
+```
+
+### Dismissing
+
+The `dismissible` method is not available for custom notices. If you want to add a dismiss button,
+you will need to do so manually. Fortunately, there is a simple way to do this.
+
+```php
+use StellarWP\AdminNotices\AdminNotices;
+use StellarWP\AdminNotices\AdminNotice;
+use \StellarWP\AdminNotices\DataTransferObjects\NoticeElementProperties;
+
+$renderCallback = function (AdminNotice $notice, NoticeElementProperties $elements) {
+    return "
+        <div>
+            <p>This is a custom notice</p>
+            <button type='button' {$elements->customCloserAttributes()}>
+                <span class='screen-reader-text'>Dismiss this notice.</span>
+            </button>
+        </div>
+    ";
+};
+
+AdminNotices::show('my_notice', $renderCallback)
+    ->custom();
+```
+
+The [$elements](src/DataTransferObjects/NoticeElementProperties.php) object provides a
+`customCloserAttributes` method that returns the necessary attributes to be place on the dismiss
+button. This method will add the necessary attributes to dismiss the notice when clicked. This will
+permanently dismiss the notice, and fade out the notice, similar to the standard WordPress
+dismissible notices.
+
+If you want the notice to be marked as dismissed, but not fade out, you can pass "clear" to the
+`customerCloserAttributes` method: e.g. `$elements->customCloserAttributes('clear')`.
```

Comment:
The word `clear` is a bit generic. Maybe `no-fade` or `immediate` instead of `clear`?

Reply:
@borkweb My thinking here was to indicate what it's doing rather than what it's not doing. I agree that the word "clear" isn't great. What we're doing, in more words, is marking the notice as dismissed for that user.

Maybe "mark-dismissed"?

Reply:
Tentatively resolved in 3885a22